app/client/src/components/DropZone.vue

```diff
@@ -2,8 +2,9 @@
   <div>
     <div v-bind='getRootProps()' class=dropzone>
       <input v-bind='getInputProps()' />
-      <p v-if='isDragActive'>Drop the files here ...</p>
-      <p v-else>Drag and drop your fasta files here, or click to select files</p>
+      <p v-if='isDragActive' class="dropzone-text">Drop the files here ...</p>
+      <p v-else class="dropzone-text">Drag and drop your fasta files here,
+        or click to select files</p>
```

Comment:
I guess this change of text is the reason for the width change on drag over, should just need some css to make fixed width.